src/app/helpers/announcement/ObjectStore/ObjectStoreRest.js

```diff
@@ -80,63 +87,41 @@ class ObjectStoreRest extends ObjectStore {
       }
     })
   }
-
-  getObjectById(data) {
-    return this.__getObjectById()(data)
-  }
-
-  __getObjectById() {
-    return async((data) => {
-      await (this.validateGetObjectById(data))
-      return await (this.findObject({ table: data.table, query: { id: data.id } }))
-    })
-  }
-
   updateObjectById(data) {
     return this.__updateObjectById()(data)
   }
 
     __updateObjectById() {
         return async((data) => {
-            await (this.validateUpdateObjectById(data))
-            let options = {
-                method: 'POST',
-                uri: envVariables.DATASERVICE_URL + 'data/v1/object/update',
-                body: {
-                    request: {
-                        'tableName': data.table,
-                        'documentName': data.table,
-                        'payload': data.values
-                    }
-                },
-                json: true
-            }
             try {
+                if (!data.values) throw {msg: 'Values required to update', status: 'error'}
+                if (!data.values.id) throw {msg: 'Id should be of type string!', status: 'error'}
+                let options = {
+                    method: 'POST',
+                    uri: envVariables.DATASERVICE_URL + 'data/v1/object/update',
+                    body: {
+                        request: {
+                            'tableName': data.table,
+                            'documentName': data.table,
+                            'payload': data.values
+                        }
+                    },
+                    json: true
+                }
                 let result = await (this.httpService(options))
```

Comment:
why does this use its own httpservice? why not use the http wrapper which is used in the notification service also?

Reply:
This is not in the part of notification service we have done this clean up in announcement model service task

Reply:
great.